packages/core/ios/Sources/QueueHolder.swift

```diff
@@ -0,0 +1,11 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2016-Present Datadog, Inc.
+ */
+
+import Foundation
+
+internal struct QueueHolder {
+    internal static let queue = dispatch_queue_serial_t(label: "dd-react-native-sdk")
+}
```

Comment:
this is okay but you could also declare a global `sharedQueue`
```swift
// at top-level, not in any class/struct/enum scope
internal let sharedQueue = DispatchQueue(label: "some label")
```
then you can use it from everywhere in our module

Reply:
thanks for the suggestion! implemented.